### PR TITLE
Move DetectCoral into the Axon repository

### PR DESCRIPTION
**Pull request by GrantPerkins**
This commit includes GitHub Action Docker builds for automatically pushing to the proper DockerHub repositories.

### CODE REVIEW

File: packages/ml/mobilenet/training/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM wpilib/axon-base:latest
+
+RUN mkdir -p /tensorflow/models/research/learn && \
+    mkdir -p /tensorflow/models/research/learn/ckpt && \
+    rm -rf /tensorflow/models/research/learn/train && \
+    rm -rf /tensorflow/models/research/learn/models && \
+    mkdir -p /tensorflow/models/research/start_ckpt  && \
+    cd /tensorflow/models/research/start_ckpt && \
+    wget "https://github.com/wpilibsuite/DetectCoral/releases/download/v2/model.ckpt.data-00000-of-00001" && \
```
Comment (GrantPerkins) [inline packages/ml/mobilenet/training/Dockerfile]: Not sure where to host this.

Comment (NLZ432) [reply packages/ml/mobilenet/training/Dockerfile]: I guess we could do it here right?

Comment (GrantPerkins) [reply packages/ml/mobilenet/training/Dockerfile]: Hosting in a release is quirky. Maybe wpilib has cloud storage somewhere

**Comment by GrantPerkins** [comment]
Expecting this to fail lint, as I have comments in my .yml. Going to remove once push to axon-base succeeds

**Review by NLZ432 — APPROVED** [review]
nice

**Pull request merged**